src/getCoinCombination.js

```diff
@@ -6,13 +6,15 @@
  * @returns {number[]}
  */
 function getCoinCombination(cents) {
-  let currentAmount = cents;
+  let currentAmount = Math.trunc(cents);
   const values = [1, 5, 10, 25];
   const coins = [0, 0, 0, 0];
 
-  for (let i = 3; i >= 0; i--) {
-    coins[i] = Math.floor(currentAmount / values[i]);
-    currentAmount -= coins[i] * values[i];
+  if (cents >= 0) {
```

Comment:
The condition `if (cents >= 0)` is correctly placed to ensure that the function only processes non-negative values. However, consider adding a check to handle negative inputs explicitly, such as returning an empty array or throwing an error, as the current implementation will return an array of zeros for negative inputs.

Reply:
changed to an error